src/org/jitsi/impl/neomedia/MediaUtils.java

```diff
@@ -366,12 +367,20 @@
 
         for (MediaFormat[] mediaFormats
                 : rtpPayloadTypeStrToMediaFormats.values())
+        {
             for (MediaFormat mediaFormat : mediaFormats)
+            {
                 if (MediaType.AUDIO.equals(mediaFormat.getMediaType()))
+                {
                     audioMediaFormats.add(mediaFormat);
+                }
+            }
+        }
         for (MediaFormat mediaFormat : rtpPayloadTypelessMediaFormats)
+        {
             if (MediaType.AUDIO.equals(mediaFormat.getMediaType()))
                 audioMediaFormats.add(mediaFormat);
```

Comment:
It looks inconsistent to me to add all {} in the previous loop and not add all in the next loop.